main.js

```diff
@@ -0,0 +1,43 @@
+var items = [
+  {name: 'Audio-Technica ADTH-AD700' , price: 178.90 , img: "/Users/christophervoss/Desktop/headphone.jpg"},
+  {name: 'Cowin E-7 w/ Acive Noise Cancelling + Bluetooth', price: 65, img: 'https://images-na.ssl-images-amazon.com/images/I/619jaB5tJZL._SL1001_.jpg'},
+  {name: 'Bose QuietComfort 25 Acoustic Noise Cancelling Headphones', price: 299, img: 'https://images-na.ssl-images-amazon.com/images/I/71kgntub7mL._SL1500_.jpg'},
+  {name: 'Sennheiser HD 700', price: 598, img: 'https://images-na.ssl-images-amazon.com/images/I/41XoulU95pL.jpg'}
+]
+
+var $row = document.querySelector('div.row')
+
+function currentItem(itemList) {
+  for (var i = 0; i < itemList.length; i++) {
+    var currentProduct = itemList[i]
+    var $product = displayItem(currentProduct)
+    $row.appendChild($product)
+  }
+}
+
+function displayItem(product) {
```

Comment:
we're not displaying here, just returning some DOM. Perhaps a better name would be `renderItem`